### PR TITLE
PA 1.3.0 release notes

### PR DESCRIPTION
**Pull request by MariekeGO**
@edalslc, could you check if this is OK to merge?

### CODE REVIEW

File: release-notes/PA/PA_1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+| Skyline Process Automation Gateway | Repeat Gateway |
+
+When you run the *SRM_Setup* script, the packages will be imported.
+
+The setup script will now also create a *Generic Gateway* element using the *Skyline Process Automation Gateway* connector, including specific resource pools for both Generic Gateway and Repeat Gateway functions.
+
+> [!IMPORTANT]
+> When you update an existing installation with a new package, run the *SRM_Setup* script to make sure you are using the most recent version of the function packages.
+
+#### Integration of Skyline Process Automation Overview functions [ID_30496] [ID_30764]
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.0.md:29]: Integration of Skyline Process Automation and associated functions

File: release-notes/PA/PA_1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+
+The Process Automation Framework has been adjusted to support the following *Skyline Process Automation Overview* functions:
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.0.md:31]: I would remove the reference to the Skyline Process Automation Overview

File: release-notes/PA/PA_1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,478 @@
...
+> [!NOTE]
+> In case these functions are used, the service definition must have a property *ServiceDefinitionType* with value "ProcessAutomation".
+
+For the *PA Timer Start Event* function, the following parameters have been added:
+
+- *PA Timer Start Event Active End Date*: The date when the execution of an event can stop, in the format dd/mm/yyyy.
+- *PA Timer Start Event Active End Time*: The time on any day between *PA Timer Start Event Active Start Date* and *PA Timer Start Event Active End Date* when the event stops executing, in the format HH:MM.
+- *PA Timer Start Event Active Start Date*: The date when the execution of an event can begin, in the format dd/mm/yyyy.
+- *PA Timer Start Event Active Start Time*: The time on any day between *PA Timer Start Event Active Start Date* and *PA Timer Start Event Active End Date* when the event begins executing, in the format HH:MM.
+- *PA Timer Start Event Frequency Interval*: The days when the event is executed. This depends on the value of *PA Timer Start Event Frequency Type*.
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.0.md:50]: parameters have been replaced with "PA Timer Start Recurring Pattern" ( see RN30764)

File: release-notes/PA/PA_1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,478 @@
...
+
+The LSO script that runs when the activation window starts has also been extended to validate all user tasks. It will check if the DOM definition is correctly configured and will notify the user in case of an invalid or missing configuration.
+
+#### User task process definition configuration [ID_33111]
+
+It is now possible to configure user tasks in the process definition configuration UI of the Process Automation Framework. To do so, click the *Configure* button next to a user task activity in the process definition configuration script. You will then be able to configure the user task based on existing profile instances or create new profile instances based on the existing user task DOM definition.
+
+#### Wizard to configure a process [ID_33174]
+
+A wizard is now available that allows you to configure a process. It allows you to create, configure, and activate a process definition, and it provides an overview of the active and future processes with their activation windows, as well as an overview of all active tokens.
```
Comment (edalslc) [inline release-notes/PA/PA_1.3.0.md:270]: App

**Comment by MariekeGO** [comment]
@edalslc, I've made the necessary changes. Could you take a look?

**Review by edalslc — APPROVED** [review]

**Pull request merged**